src/components/objectListing/objectListing.component.tsx

```diff
@@ -113,24 +111,24 @@ const createColumns = (
   });
 
   // TODO only add/create this column if the schema has images. Or always created it but hide it when it doesn't have images
-  const imagesColumn = columnHelper.accessor("images", {
-    header: formatObjectField("Images"),
-    cell: (props) => {
-      const images = props.getValue<SkylarkGraphQLObjectImage[]>();
-      if (!images || images.length === 0) {
-        return "";
-      }
-
-      return (
-        <div>
-          {images.map(({ uid, url, title }) => (
-            // eslint-disable-next-line @next/next/no-img-element
-            <img src={url} key={`${props.row.id}-${uid}`} alt={title} />
-          ))}
-        </div>
-      );
-    },
-  });
+  // const imagesColumn = columnHelper.accessor("images", {
```

Comment:
we want to keep this code?

Reply:
Yeah its just to speed up stuff at the moment